src/main/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStaging.java

```diff
@@ -31,6 +31,9 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
+import java.util.StringTokenizer;
```

Comment:
StringTokenizer is a legacy class that is retained for compatibility reasons although its use is discouraged in new code.

Reply:
ah gotcha, will switch it to something else